homeassistant/const.py

```diff
@@ -240,10 +241,18 @@
 DEVICE_CLASS_HUMIDITY: Final = "humidity"
 DEVICE_CLASS_ILLUMINANCE: Final = "illuminance"
 DEVICE_CLASS_MONETARY: Final = "monetary"
+DEVICE_CLASS_NITROGEN_DIOXIDE = "nitrogen_dioxide"
+DEVICE_CLASS_NITROGEN_MONOXIDE = "nitrogen_monoxide"
+DEVICE_CLASS_NITROUS_OXIDE = "nitrous_oxide"
+DEVICE_CLASS_OZONE: Final = "ozone"
 DEVICE_CLASS_POWER_FACTOR: Final = "power_factor"
 DEVICE_CLASS_POWER: Final = "power"
+DEVICE_CLASS_PM25: Final = "pm25"
+DEVICE_CLASS_PM1: Final = "pm1"
+DEVICE_CLASS_PM10: Final = "pm10"
 DEVICE_CLASS_PRESSURE: Final = "pressure"
 DEVICE_CLASS_SIGNAL_STRENGTH: Final = "signal_strength"
+DEVICE_CLASS_SULPHUR_DIOXIDE = "sulphur_dioxide"
```

Comment:
Just marking this as todo so we don't merge by accident.

Reply:
Should all be there in https://github.com/home-assistant/core/pull/54075/commits/647799c9b1f156e27127dee5eccb9c18bf784998 hopefully